mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping

```diff
@@ -0,0 +1,53 @@
+CLASS net/minecraft/class_8985 net/minecraft/client/render/debug/BreezeDebugRenderer
+	FIELD field_47465 PINK I
+	FIELD field_47466 LIGHT_BLUE I
+	FIELD field_47467 GREEN I
+	FIELD field_47468 ORANGE I
+	FIELD field_47469 RED I
+	FIELD field_47472 client Lnet/minecraft/class_310;
+	FIELD field_47473 breezeDebugInfoList Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_55255 clearMap ()V
+	METHOD method_55256 draw (ILorg/joml/Matrix4f;DDDLnet/minecraft/class_4588;Lnet/minecraft/class_243;FI)V
+		ARG 0 vertex
+		ARG 1 matrix4f
+		ARG 8 vertexConsumer
+		ARG 9 entityBox
+	METHOD method_55257 (Lnet/minecraft/class_1297;)Lnet/minecraft/class_243;
+		ARG 1 entity
+	METHOD method_55258 (Lnet/minecraft/class_8949;)Z
+		ARG 0 predicate
+	METHOD method_55259 renderOutlines (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDD)V
+		ARG 1 stack
+		ARG 2 vertexConsumerProvider
+		ARG 3 x
+		ARG 5 y
+		ARG 7 z
+	METHOD method_55260 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_243;)V
+		ARG 9 entityVec3
+	METHOD method_55261 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_2338;)V
+		ARG 9 targetVec4
+			COMMENT targetVec4
+	METHOD method_55262 constructVertex (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_243;Lnet/minecraft/class_243;I)V
+		ARG 0 stack
+		ARG 1 vertexConsumerProvider
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 vec3
+		ARG 9 vec4
```

Comment:
```suggestion
	METHOD method_55262 drawLine (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_243;Lnet/minecraft/class_243;I)V
		ARG 0 matrices
		ARG 1 vertexConsumers
		ARG 2 cameraX
		ARG 4 cameraY
		ARG 6 cameraZ
		ARG 8 entityPos
		ARG 9 targetPos
```

It's always a good idea to check other classes for naming conventions. Also, never name something "vec3" or "vec4". Vec4 is a vector of 4 values.